packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -183,632 +166,6 @@ describe("Garbage Collection Tests", () => {
 		sessionStorageConfigProvider.value.getRawConfig = oldRawConfig;
 	});
 
-	describe("Configuration", () => {
```

Comment:
This is paranoid but consider leaving these here for this PR and then removing them in another PR (or even wait on the move til that).  Just for confidence / clear evidence that nothing regressed.

Reply:
I didn't change anything so its fine. I don't want to undo changes :-(.